src/components/ClientCard/ClientCard.jsx

```diff
@@ -20,24 +20,20 @@ const ClientCard = props => (
   </li>
 );
 
-const mapStateToProps = state => ({
-
-});
-
 
 const mapDispatchToProps = dispatch => ({
-  setActiveChat: chat => dispatch(setActiveChat(chat)),
+  setActiveChat: chatId => dispatch(setActiveChat(chatId)),
 });
 
 
 export default connect(
-  mapStateToProps,
+  () => ({}),
```

Comment:
I read from the react-redux docs that it wont re-render on redux state change if you pass it `null`, thats the only major issue and so I think you made the right decision